package.json

```diff
@@ -26,42 +26,39 @@
   },
   "homepage": "https://github.com/ConvexCooperative/baseplate-cloudflare-worker#readme",
   "devDependencies": {
-    "@babel/core": "^7.18.5",
-    "@babel/eslint-parser": "^7.18.2",
-    "@babel/preset-env": "^7.18.2",
-    "@babel/preset-typescript": "^7.17.12",
-    "@changesets/changelog-github": "^0.4.5",
-    "@changesets/cli": "^2.23.0",
-    "@cloudflare/workers-types": "^3.12.0",
-    "@jest/globals": "^28.1.1",
-    "@types/babel__core": "^7.1.19",
-    "@types/jest": "^28.1.1",
-    "@types/webpack": "^5.28.0",
-    "babel-loader": "^8.2.5",
+    "@babel/core": "^7.23.9",
+    "@babel/eslint-parser": "^7.23.10",
+    "@babel/preset-env": "^7.23.9",
+    "@babel/preset-typescript": "^7.23.3",
+    "@changesets/changelog-github": "^0.5.0",
+    "@changesets/cli": "^2.27.1",
+    "@cloudflare/workers-types": "^4.20240208.0",
+    "@jest/globals": "^29.7.0",
+    "@types/babel__core": "^7.20.5",
+    "@types/jest": "^29.5.12",
+    "@types/webpack": "^5.28.5",
     "cheerio": "1.0.0-rc.12",
     "cross-env": "^7.0.3",
-    "eslint": "^8.17.0",
+    "eslint": "^8.56.0",
     "eslint-config-ts-important-stuff": "^1.1.0",
-    "fork-ts-checker-webpack-plugin": "^7.2.11",
-    "husky": "^8.0.1",
-    "isomorphic-fetch": "^3.0.0",
-    "jest": "^28.1.1",
-    "jest-cli": "^28.1.1",
-    "jest-environment-jsdom": "^28.1.1",
+    "husky": "^9.0.11",
+    "jest": "^29.7.0",
+    "jest-cli": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
     "lodash-es": "^4.17.21",
-    "prettier": "^2.7.1",
-    "pretty-quick": "^3.1.3",
-    "typescript": "^4.7.3",
-    "wrangler": "^3.15.0"
+    "prettier": "^3.2.5",
+    "pretty-quick": "^4.0.0",
```

Comment:
pretty-quick@4 supports prettier@3, which is nice. I had to switch to lint-staged in other projects for awhile, since pretty-quick@3 didn't support prettier@3